extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -278,6 +278,14 @@ function TrackedCornerstoneViewport({
           viewportIndex={viewportIndex}
           imageIds={imageIds}
           imageIdIndex={currentImageIdIndex}
+          onNewImageDebounceTime={700}
```

Comment:
I'm mixed on using this instead of debouncing in our own app. Let's keep for now 👍